### PR TITLE
Add rudimentary build & test instructions to Readme

### LINKED ISSUE #43: Add build and test instructions
Please add basic build/test/install and deploy/usage instructions to README, so people can try out and understand better the implementation.

### PR DESCRIPTION
**Pull request by markspanbroek**
Fixes #43

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,12 @@ Join the Status community chats:
 ## Rationale
 [Nimbus: an Ethereum 2.0 Sharding Client](https://github.com/status-im/nimbus/blob/master/doc/Nimbus%20-%20An%20Ethereum%202.0%20Sharding%20Client_xt.md)
 
+## Building & Testing
+
+Install the [Nim](https://nim-lang.org) programming language. Currently Nimbus requires the latest development version of Nim, which you can install with `choosenim "#head"`.
+
+You can build the package using `nimble install`. You can run the tests using `nimble test`. And you can run the example using `nim compile --run examples/decompile_smart_contract.nim`.
```
Comment (yglukhov) [inline README.md]: Technically `nimble install` doesn't build the package, but installs the package along with its dependencies. I would just remove `nimble install` step, as `nimble test` should work fine right after git clone, and will also install the dependencies.

File: README.md [hunk 2]
```diff
@@ -13,6 +13,12 @@ Join the Status community chats:
 ## Rationale
 [Nimbus: an Ethereum 2.0 Sharding Client](https://github.com/status-im/nimbus/blob/master/doc/Nimbus%20-%20An%20Ethereum%202.0%20Sharding%20Client_xt.md)
 
+## Building & Testing
+
+Install the [Nim](https://nim-lang.org) programming language. Currently Nimbus requires the latest development version of Nim, which you can install with `choosenim "#head"`.
```
Comment (yglukhov) [inline README.md]: I'd say install latest devel either by following the installation steps at https://github.com/nim-lang/Nim or by using choosenim (add an url to choosenim).

File: README.md (hunk 1 above)
Comment (zah) [reply README.md]: There is also `nimble build` for building without installing.

**Comment by markspanbroek** [comment]
Thanks @yglukhov & @zah for the comments. I have pushed an update using your suggestions.

**Comment by yglukhov** [comment]
Thank you!

**Pull request merged**